Mutagen.Bethesda.Analyzers.Skyrim/Record/ConditionAnalyzer.cs

```diff
@@ -27,17 +27,20 @@ public void AnalyzeRecord(IsolatedRecordAnalyzerParams<ISkyrimMajorRecordGetter>
             {
                 case IGetEventDataConditionDataGetter getEventData:
                     param.AddTopic(
-                        InvalidConditionReference.Format(getEventData.Function.ToString()),
-                        x => x);
+                        InvalidConditionReference.Format(getEventData.Function.ToString()));
                     break;
                 case {} conditionData:
                     param.AddTopic(
-                        InvalidConditionReference.Format(conditionData.Function.ToString()),
-                        x => x);
+                        InvalidConditionReference.Format(conditionData.Function.ToString()));
                     break;
             }
         }
     }
+
+    public IEnumerable<Func<ISkyrimMajorRecordGetter, object?>> FieldsOfInterest()
+    {
+        yield break;
```

Comment:
We'll have to get back to this later when we have a good way to deal with conditions